app/screens/send-bitcoin-screen/send-bitcoin-confirmation-screen.tsx

```diff
@@ -219,6 +220,13 @@ const SendBitcoinConfirmationScreen: React.FC<Props> = ({ route }) => {
 
   const [paymentError, setPaymentError] = useState<string | undefined>(undefined)
   const { LL } = useI18nContext()
+  const { error } = useQuery(OnChainTxFeeDocument, {
```

Comment:
- we don't want to query `useOnChainTxFeeLazyQuery` again, this will overload the server for now reason. we should be using the result from the existing `useOnChainTxFeeLazyQuery` query
- we should not use useQuery but always use the associated types query, ie: `useOnChainTxFeeLazyQuery` or `useOnChainTxFeeQuery`

Reply:
Okay, got it!